Creiwry/kitten_project
# Front signin sign up

Ticket Link
(Found on Trello in your ticket description)
https://trello.com/c/sNHXVUTS

User Story
(Linked to on Trello in your ticket description)
https://trello.com/c/JMOoUKw2

What does this PR do?
(Your answer here.)
provides tentative front end for user registration page (CSS still needs work and JS to be added)

Where should the reviewer start?
(Your answer here.)
Perhaps having a look at manifest.js. The rest is very straightforward CSS.

How should this be manually tested?
N/A

Any background context you want to provide?
If time later - will add animation and change color of filled checkbox.

Screenshots (if appropriate)
N.A

Are there any breaking changes?
The manifest.js was updated with new stylesheet as it was not working without. this causes no problems locally but should be flagged.
